Translation page (#2224)
privacyguides/privacyguides.org: docs/meta/translations.md

---
title: Translations
---

Crowdin has good documentation and we suggest looking at their [Getting Started](https://support.crowdin.com/crowdin-intro/) guide. Our site is in [Markdown](https://en.wikipedia.org/wiki/Markdown), so it should be easy to chip in.

> # Admonitions

Reviewer: ```suggestion
## Admonitions
```

> Throughout the site we use MkDocs's [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage), to show information to readers about the products such as `example`, `warning`, `tip`, etc.

Reviewer: ```suggestion
Throughout the site we use MkDocs's [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage), to show information to readers. They come in a few different flavors such as `example`, `warning`, `tip`, etc.
```

> By default when admonitions are used they will have an English string on the site. This can be [customized](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title), without too much effort. For example if you were translating and admonition of type [warning](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#type:warning) to Dutch, this is how you would write it:

Reviewer: ```suggestion
By default when admonitions are used they will have an English string on the site. This can be [customized](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title), without too much effort. For example, if you were translating an admonition of type [`warning`](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#type:warning) to Dutch, this is how you would write it:
```

```text
!!! warning "Waarschuwing"
```

> Downloads are a [custom admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#custom-admonitions) that we use and you would write that like:

Reviewer: ```suggestion
Downloads are a [custom admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#custom-admonitions) which is written as follows:
```

```text
??? downloads "Downloaden"
```

> The same goes for other types, such as `tip`, `example`, `warning`, `danger` etc. Recommendations are also admonitions, but they do not need overriding, because the default has no text, so they are always:

Reviewer: ```suggestion
The same goes for other types, such as `tip`, `example`, `warning`, `danger` etc. Recommendations are also admonitions, but they do not need overriding. Because the default has no text, they are always written like this:
```

Author: I was always taught to never start a sentence with because

Reviewer: It's fine in this [case](https://www.quickanddirtytips.com/articles/can-you-start-a-sentence-with-because/) since there's an independent clause following it.

Author: I think I re worded this so it won't be a fragment.

```text
!!! recommendation
```

> # Translation output

Reviewer: ```suggestion
## Translation output
```

> Translation software gets the translation quite reliable however, attention does need to be given that the translated string is correct.

Reviewer: ```suggestion
Translation software gets the translation quite accurate; however, you need to make sure translated string is correct.
```

For example:

```text
![Software logo](assets/img/path/to/image.svg){ align=right }
```

> We have sometimes found that the syntax for inserting an image like above was missing the `![` or an extra space was placed between the text and the path, eg `](`. If a translation string is clearly not correct, we encourage you to **delete** it by pressing the trash icon [or vote](https://support.crowdin.com/enterprise/getting-started-for-volunteers/#voting-view) which one you think sounds best. When invalid strings are deleted they are removed from the organization's [translation memory](https://support.crowdin.com/enterprise/translation-memory), meaning that when the source string is seen again, it won't suggest the incorrect translation.

Reviewer: ```suggestion
We have sometimes found that the syntax for inserting an image like above was missing the `![` or an extra space was placed between the text and the path, e.g. `](`. If a translation string is clearly not correct, we encourage you to **delete** it by pressing the trash icon [or vote](https://support.crowdin.com/enterprise/getting-started-for-volunteers/#voting-view) on which one you think sounds best. When invalid strings are deleted, they are removed from the organization's [translation memory](https://support.crowdin.com/enterprise/translation-memory), meaning that when the source string is seen again, it won't suggest the incorrect translation.
```

> We'd like to thank the [translation team](https://crowdin.com/project/privacyguides/reports/top-members) who spent many hours on translating the content, that we now have. We're going to launch in Dutch, French and Hebrew.

Reviewer: ```suggestion
We'd like to thank the [translation team](https://crowdin.com/project/privacyguides/reports/top-members) who spent many hours on translating the content of the site. We're going to launch in Dutch, French and Hebrew.
```

Reviewer 2: No need to include ”timely” information from the blog post like “we’re going to launch…” etc., this meta page should strictly be for translation policies/guides.

Author: Yes it should be that might have been my mistake from copying it over from the blog article.

# Punctuation

> For examples like the above admonitions inverted commas, eg: `" "` must be used to specify string text. MkDocs will not correctly interpret other symbols ie, `「 」` or `« »`. They work fine within the text, for the translation though.

Reviewer: ```suggestion
For examples like the above admonitions quotation marks, eg: `" "` must be used to specify string text. MkDocs will not correctly interpret other symbols ie, `「 」` or `« »`. They work fine within the text, for the translation though.
```
Americans call those quotation marks, dunno if it matters too much but we agreed on American English.

Reviewer 2: This section is also a bit confusing. Does this **only** apply to admonitions (I think so?) or are there other areas where MkDocs is interpreting things as strings? If this is only for admonitions it should be moved to that section I think, or can you give another example of where this applies?

Author: Only admonitions, because they are customizable by translators and need to remain so.

> # Brackets

Reviewer: ```suggestion
## Brackets
```

> For links these must regular brackets ie `( )` CJK style brackets such as `（ ）` will not work for links.

Reviewer: ```suggestion
For links, these must regular brackets i.e. `( )`. CJK style brackets such as `（ ）` will not work for links.
```

### Support

Feel free to check out our localization room on Matrix [#pg-i18n:aragon.sh](https://matrix.to/#/%23pg-i18n:aragon.sh) if you have any questions on getting started.

> Please note that the English version of the site is the primary version, meaning changes occur there first. This means it is still possible that specific languages may be behind. If you notice such an instance please help out. We cannot guarantee the accuracy of all our translations. If you have a suggestion about content specific to your region, please open an issue or pull request to our [main repository](https://github.com/privacyguides/privacyguides.org).

Reviewer: ```suggestion
Please note that the English version of the site is the primary version, meaning changes occur there first. If you notice a language falling behind the English version, please help out. We cannot guarantee the accuracy of all our translations. If you have a suggestion about content specific to your region, please open an issue or pull request to our [main repository](https://github.com/privacyguides/privacyguides.org).
```
